package.json

```
@@ -5,7 +5,8 @@
   "main": "index.ts",
   "scripts": {
     "build": "tsc",
-    "test": "jest"
+    "test": "jest",
+    "prettier": "prettier --write src"
```

Maybe we could have a check also.

```suggestion
    "prettier": "prettier src",
    "prettier:fix": "prettier --write src"
```